QUARKUS-167.md

```diff
@@ -0,0 +1,39 @@
+# Quarkus-190 - Testing and verifying Quarkus/Vertx SQL
+
+The goal of this test plan is to cover the main points of conflicts between Quarkus/Vertx and SQL databases. 
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-190 
+
+
+## Scope of the testing
+Following actions were taken to ensure test coverage and automation for vertx/sql applications:
+ - Setup databases and load initial data through Flyway
+ - Programmatically insert statements
+ - Select statements
+ - Transactional statements between several tables
+ - Supported databases: postgresql, mysql/mariaDB, DB2
+
+### Impact on testsuites and testing automation:
+ - Ensure this coverage works both in JVM and NATIVE mode
+ - Integration test it's cover by Beefy-scenarios testsuite
+ - Some Openshift integration tests as Db2 are pending. 
+
+### Impact on resources:
```

Comment:
MySQL / PostgreSQL / DB2

Reply:
The intention of this comment was to change
```
 - mysql (183.6 MiB - 343.5 MiB)
 - postgresql (9.9 MiB)
 - db2 (115.7Mb)
```
to
```
 - MySQL (183.6 MiB - 343.5 MiB)
 - PostgreSQL (9.9 MiB)
 - DB2 (115.7Mb)
 ```

Why does MySQL have such diff (183.6 MiB - 343.5 MiB) in sizes? I assume it's about image size.
10 MiB for PostgreSQL is suspicious